### DISCUSSION TITLE
Be on the lookout for shady connections

### DISCUSSION THREAD

**Discussion by lpaln** [Chia-Network/chia-blockchain#8184] [General]
Not sure what this is about, but I guess I have to add it to my list of monitoring parameters

![snip](https://user-images.githubusercontent.com/80719229/130530815-20d393ee-5b3b-413d-8fa2-87d63e61e0fa.JPG)

always followed by:
_WARNING  Invalid handshake with peer. Maybe the peer is running old software._

The IP traces back to a Ukraine ISP. quickly checked and 8444 is closed on this host and seems to silently drop incoming traffic. I may be being paranoid, but I'm blocking this entire subnet in my firewall.

Chia client should automatically ban these after a couple attempts. I think this activity looks reasonably suspicious to me.

**Comment by loppefaaret** [comment]
there is the possibility that it is a misconfigured client from a forked project

**Comment by BrandtH22** [comment]
@lpaln , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
